ArduPlane/Parameters.cpp

```diff
@@ -1177,6 +1177,12 @@ const AP_Param::GroupInfo ParametersG2::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("FLIGHT_OPTIONS", 13, ParametersG2, flight_options, 0),
 
+#ifdef ENABLE_SCRIPTING
```

Comment:
Shouldn't this be `SCRIPTING_ENABLED` instead?

Reply:
There are both in the code base, I find that `ENABLE_` first reads better, particularly when it's used in a ifdef rather then a comparison like this. I'm inclined to leave it unless there is strong objection from others.

Reply:
On https://github.com/ArduPilot/ardupilot/blob/master/ArduCopter/APM_Config.h I only find `*_ENABLED` patterns, not a single `ENABLE_*`